gulpfile.js

```diff
@@ -39,11 +42,25 @@ const babelCompile = () => gulp.src(['src/**/*.js'])
  *
  * @returns {Stream}
  */
-const lint = () => gulp.src(['src/**/*.js', '*.js'])
+const lint = () => gulp.src(lintGlobs)
   .pipe(eslint())
   .pipe(eslint.format())
   .pipe(eslint.failAfterError());
 
+const isFixed = file => file.eslint != null && file.eslint.fixed;
```

Comment:
Linting error:

```
  50:17  error  Expected parentheses around arrow function argument  arrow-parens

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.
```

Also, `file.eslint != null` could be improved by using the fact of falsy value.